tasks/base64-encode-checksum/README.md

```diff
@@ -0,0 +1,10 @@
+# base64-encode-checksum
+
+Task to encode the Terraform Checksum file
+
+## Parameters
+
+| Name                 | Description                                                        | Optional  | Default value                                         |
+|----------------------|--------------------------------------------------------------------|-----------|-------------------------------------------------------|
+| subdirectory   | Subdirectory inside the workspace to be used                             | Yes       | ""                                                    |
```

Comment:
can you fix the formatting here? Looks like everything is aligned vertically except for the `|` after subdirectory

Reply:
Fixed